ubuntu/debian/control

```diff
@@ -121,11 +122,11 @@ Description: Ignition Common classes and functions (Events) - Shared library
 Package: libignition-common5-events-dev
 Architecture: any
 Section: libdevel
-Depends: libignition-cmake2-dev,
-         libignition-common5-core-dev (= ${binary:Version}),
+Depends: libgdal-dev,
+         libignition-cmake2-dev,
+         libignition-common5-graphics-dev (= ${binary:Version}),
```

Comment:
should these be added under the `events` component?

Reply:
should be fixed by https://github.com/ignition-release/ign-common5-release/pull/4/commits/8555fbca8488c6f79d8b2464d0bfa29827e8ceda